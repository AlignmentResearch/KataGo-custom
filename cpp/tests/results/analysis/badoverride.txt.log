: Analysis Engine starting...
: KataGo v1.10.0
: nnRandSeed0 = analysisTest
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 allowing smaller boards
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded config configs/analysis_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logAllRequests = true
: Config override: logAllResponses = true
: Config override: logDir = 
: Config override: logFile = tests/results/analysis/badoverride.txt.log
: Config override: logSearchInfo = true
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = analysisTest
: Config override: nnRandomize = false
: Config override: numAnalysisThreads = 1
: Config override: numSearchThreadsPerAnalysisThread = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: trtUseFP16 = false
: Analyzing up to 1 positions at at time in parallel
: Started, ready to begin handling requests
: Request: {"id":"badoverride","initialStones":[],"moves":[],"rules":"tromp-taylor","komi":7.5,"boardXSize":19,"boardYSize":19,"overrideSettings":{"sldjfslkjf":1234}}
: Response: {"field":"overrideSettings","id":"badoverride","warning":"Unknown config params: sldjfslkjf"}
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreAREAtaxNONEsui1komi7.5
Root visits: 100
New playouts: 100
NN rows: 91
NN batches: 91
NN avg batch size: 1
PV: Q17 R4 D3 D17
Tree:
: T   0.22c W  -0.46c S   0.24c ( +0.0 L  +0.1) N     100  --  Q17 R4 D3 D17
---Black(^)---
Q17 : T   1.77c W   0.65c S   0.43c ( +0.3 L  +0.3) LCB   -3.80c P  8.73% WF  12.1 PSV      17 N      12  --  Q17 R4 D3 D17
D3  : T   1.34c W   0.48c S   0.43c ( +0.3 L  +0.4) LCB   -7.09c P  8.43% WF  10.1 PSV      10 N      10  --  D3 R16 C16 Q3
C4  : T   0.77c W  -0.03c S   0.29c ( +0.1 L  +0.1) LCB   -8.97c P  8.60% WF  10.0 PSV       8 N      10  --  C4 R16 C16 Q3
C16 : T   0.84c W  -0.20c S   0.34c ( +0.2 L  +0.3) LCB   -7.98c P  8.21% WF  10.0 PSV       8 N      10  --  C16 Q3 C4 Q17
Q3  : T   0.46c W  -0.10c S   0.38c ( +0.2 L  +0.3) LCB   -9.02c P  8.99% WF   9.0 PSV       8 N       9  --  Q3 R16 D3 D17
R4  : T  -0.27c W  -0.67c S   0.19c ( -0.1 L  +0.1) LCB   -8.41c P  8.90% WF   9.9 PSV       6 N      10  --  R4 R16 D3 D17
D17 : T  -0.15c W  -0.86c S   0.17c ( -0.1 L  +0.0) LCB  -26.67c P  8.00% WF   6.0 PSV       6 N       6  --  D17 D3 Q3 Q17
R16 : T   0.72c W  -0.10c S   0.28c ( +0.1 L  +0.1) LCB  -20.16c P  7.34% WF   6.0 PSV       6 N       6  --  R16 Q3 C4 D17
D4  : T  -0.71c W  -1.14c S   0.15c ( -0.1 L  -0.0) LCB  -25.68c P  4.67% WF   6.0 PSV       3 N       6  --  D4 R16 C16 Q3
Q16 : T  -0.12c W  -0.96c S   0.12c ( -0.2 L  -0.1) LCB  -29.55c P  3.90% WF   5.0 PSV       3 N       5  --  Q16 D3 D17

: Response: {"id":"badoverride","isDuringSearch":false,"moveInfos":[{"lcb":0.483356084,"move":"Q17","order":0,"prior":0.0873461664,"pv":["Q17","R4","D3","D17"],"scoreLead":0.285215055,"scoreMean":0.285215055,"scoreSelfplay":0.306614362,"scoreStdev":30.5786629,"utility":0.0176545712,"utilityLcb":-0.0380412688,"visits":12,"winrate":0.503247456},{"lcb":0.472327803,"move":"D3","order":1,"prior":0.0843356103,"pv":["D3","R16","C16","Q3"],"scoreLead":0.356990204,"scoreMean":0.356990204,"scoreSelfplay":0.301339342,"scoreStdev":30.5359403,"utility":0.0133850569,"utilityLcb":-0.0708517355,"visits":10,"winrate":0.502412372},{"lcb":0.465085789,"move":"C4","order":2,"prior":0.0860430747,"pv":["C4","R16","C16","Q3"],"scoreLead":0.118244064,"scoreMean":0.118244064,"scoreSelfplay":0.0924139681,"scoreStdev":30.530926,"utility":0.00766767375,"utilityLcb":-0.0897161767,"visits":10,"winrate":0.499865736},{"lcb":0.467496253,"move":"C16","order":3,"prior":0.0821324214,"pv":["C16","Q3","C4","Q17"],"scoreLead":0.282250769,"scoreMean":0.282250769,"scoreSelfplay":0.172336918,"scoreStdev":30.4562836,"utility":0.00835345947,"utilityLcb":-0.0798221765,"visits":10,"winrate":0.498987551},{"lcb":0.465667447,"move":"Q3","order":4,"prior":0.0898634866,"pv":["Q3","R16","D3","D17"],"scoreLead":0.349845593,"scoreMean":0.349845593,"scoreSelfplay":0.224142058,"scoreStdev":30.4978388,"utility":0.00455703539,"utilityLcb":-0.0901633365,"visits":9,"winrate":0.499496151},{"lcb":0.467571048,"move":"R4","order":5,"prior":0.0889517292,"pv":["R4","R16","D3","D17"],"scoreLead":0.0718897181,"scoreMean":0.0718897181,"scoreSelfplay":-0.0534601679,"scoreStdev":30.578526,"utility":-0.00272730238,"utilityLcb":-0.0841158307,"visits":10,"winrate":0.49663838},{"lcb":0.400985286,"move":"D17","order":6,"prior":0.0799881965,"pv":["D17","D3","Q3","Q17"],"scoreLead":0.00714592532,"scoreMean":0.00714592532,"scoreSelfplay":-0.0752544496,"scoreStdev":30.6490749,"utility":-0.00152339147,"utilityLcb":-0.266659931,"visits":6,"winrate":0.495676907},{"lcb":0.424919717,"move":"R16","order":7,"prior":0.0734395757,"pv":["R16","Q3","C4","D17"],"scoreLead":0.0842029773,"scoreMean":0.0842029773,"scoreSelfplay":0.0817353116,"scoreStdev":30.6050072,"utility":0.00721390361,"utilityLcb":-0.201571448,"visits":6,"winrate":0.499485914},{"lcb":0.405129554,"move":"D4","order":8,"prior":0.0467210226,"pv":["D4","R16","C16","Q3"],"scoreLead":-0.00165091884,"scoreMean":-0.00165091884,"scoreSelfplay":-0.110294712,"scoreStdev":30.6087464,"utility":-0.00712438763,"utilityLcb":-0.256753578,"visits":6,"winrate":0.494282836},{"lcb":0.39007354,"move":"Q16","order":9,"prior":0.0390344262,"pv":["Q16","D3","D17"],"scoreLead":-0.122380172,"scoreMean":-0.122380172,"scoreSelfplay":-0.152055663,"scoreStdev":30.7004796,"utility":-0.00115715184,"utilityLcb":-0.295494352,"visits":5,"winrate":0.495193968},{"lcb":0.210894392,"move":"Q4","order":10,"prior":0.0582818054,"pv":["Q4","D3","D17"],"scoreLead":-0.054561992,"scoreMean":-0.054561992,"scoreSelfplay":-0.206756195,"scoreStdev":30.5874756,"utility":-0.0126349461,"utilityLcb":-0.796848313,"visits":4,"winrate":0.490970594},{"lcb":0.0932849796,"move":"D16","order":11,"prior":0.0466097519,"pv":["D16","D3","Q17"],"scoreLead":-0.370538535,"scoreMean":-0.370538535,"scoreSelfplay":-0.47277722,"scoreStdev":30.7032725,"utility":-0.0168624515,"utilityLcb":-1.12477591,"visits":4,"winrate":0.488968356},{"lcb":-0.491438841,"move":"C3","order":12,"prior":0.0324178748,"pv":["C3","R16","C16"],"scoreLead":-0.664244846,"scoreMean":-0.664244846,"scoreSelfplay":-0.482832193,"scoreStdev":30.7828552,"utility":-0.0141759727,"utilityLcb":-2.77438059,"visits":3,"winrate":0.494348521},{"lcb":-0.512627207,"move":"C17","order":13,"prior":0.0303994678,"pv":["C17","R16"],"scoreLead":-0.696721703,"scoreMean":-0.696721703,"scoreSelfplay":-0.665783852,"scoreStdev":30.7520999,"utility":-0.0275793309,"utilityLcb":-2.8,"visits":2,"winrate":0.487372793},{"lcb":-0.505080722,"move":"R3","order":14,"prior":0.0199293494,"pv":["R3","R16"],"scoreLead":-0.308975212,"scoreMean":-0.308975212,"scoreSelfplay":-0.199910849,"scoreStdev":30.7235825,"utility":-0.00930808168,"utilityLcb":-2.8,"visits":2,"winrate":0.494919278}],"rootInfo":{"currentPlayer":"B","scoreLead":0.13919092,"scoreSelfplay":0.0775131322,"scoreStdev":30.5802453,"symHash":"47E1EBDAE69A5DED3FF8DB1FA4E91845","thisHash":"47E1EBDAE69A5DED3FF8DB1FA4E91845","utility":0.00410846401,"visits":100,"winrate":0.49834412},"turnNumber":0}
: tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: NN rows: 91
: NN batches: 91
: NN avg batch size: 1
: GPU -1 finishing, processed 91 rows 91 batches
: All cleaned up, quitting
