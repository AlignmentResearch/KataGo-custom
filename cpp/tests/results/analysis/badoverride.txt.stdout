{"field":"overrideSettings","id":"badoverride","warning":"Unknown config params: sldjfslkjf"}
{"id":"badoverride","isDuringSearch":false,"moveInfos":[{"lcb":0.483356084,"move":"Q17","order":0,"prior":0.0873461664,"pv":["Q17","R4","D3","D17"],"scoreLead":0.285215055,"scoreMean":0.285215055,"scoreSelfplay":0.306614362,"scoreStdev":30.5786629,"utility":0.0176545712,"utilityLcb":-0.0380412688,"visits":12,"winrate":0.503247456},{"lcb":0.472327803,"move":"D3","order":1,"prior":0.0843356103,"pv":["D3","R16","C16","Q3"],"scoreLead":0.356990204,"scoreMean":0.356990204,"scoreSelfplay":0.301339342,"scoreStdev":30.5359403,"utility":0.0133850569,"utilityLcb":-0.0708517355,"visits":10,"winrate":0.502412372},{"lcb":0.465085789,"move":"C4","order":2,"prior":0.0860430747,"pv":["C4","R16","C16","Q3"],"scoreLead":0.118244064,"scoreMean":0.118244064,"scoreSelfplay":0.0924139681,"scoreStdev":30.530926,"utility":0.00766767375,"utilityLcb":-0.0897161767,"visits":10,"winrate":0.499865736},{"lcb":0.467496253,"move":"C16","order":3,"prior":0.0821324214,"pv":["C16","Q3","C4","Q17"],"scoreLead":0.282250769,"scoreMean":0.282250769,"scoreSelfplay":0.172336918,"scoreStdev":30.4562836,"utility":0.00835345947,"utilityLcb":-0.0798221765,"visits":10,"winrate":0.498987551},{"lcb":0.465667447,"move":"Q3","order":4,"prior":0.0898634866,"pv":["Q3","R16","D3","D17"],"scoreLead":0.349845593,"scoreMean":0.349845593,"scoreSelfplay":0.224142058,"scoreStdev":30.4978388,"utility":0.00455703539,"utilityLcb":-0.0901633365,"visits":9,"winrate":0.499496151},{"lcb":0.467571048,"move":"R4","order":5,"prior":0.0889517292,"pv":["R4","R16","D3","D17"],"scoreLead":0.0718897181,"scoreMean":0.0718897181,"scoreSelfplay":-0.0534601679,"scoreStdev":30.578526,"utility":-0.00272730238,"utilityLcb":-0.0841158307,"visits":10,"winrate":0.49663838},{"lcb":0.400985286,"move":"D17","order":6,"prior":0.0799881965,"pv":["D17","D3","Q3","Q17"],"scoreLead":0.00714592532,"scoreMean":0.00714592532,"scoreSelfplay":-0.0752544496,"scoreStdev":30.6490749,"utility":-0.00152339147,"utilityLcb":-0.266659931,"visits":6,"winrate":0.495676907},{"lcb":0.424919717,"move":"R16","order":7,"prior":0.0734395757,"pv":["R16","Q3","C4","D17"],"scoreLead":0.0842029773,"scoreMean":0.0842029773,"scoreSelfplay":0.0817353116,"scoreStdev":30.6050072,"utility":0.00721390361,"utilityLcb":-0.201571448,"visits":6,"winrate":0.499485914},{"lcb":0.405129554,"move":"D4","order":8,"prior":0.0467210226,"pv":["D4","R16","C16","Q3"],"scoreLead":-0.00165091884,"scoreMean":-0.00165091884,"scoreSelfplay":-0.110294712,"scoreStdev":30.6087464,"utility":-0.00712438763,"utilityLcb":-0.256753578,"visits":6,"winrate":0.494282836},{"lcb":0.39007354,"move":"Q16","order":9,"prior":0.0390344262,"pv":["Q16","D3","D17"],"scoreLead":-0.122380172,"scoreMean":-0.122380172,"scoreSelfplay":-0.152055663,"scoreStdev":30.7004796,"utility":-0.00115715184,"utilityLcb":-0.295494352,"visits":5,"winrate":0.495193968},{"lcb":0.210894392,"move":"Q4","order":10,"prior":0.0582818054,"pv":["Q4","D3","D17"],"scoreLead":-0.054561992,"scoreMean":-0.054561992,"scoreSelfplay":-0.206756195,"scoreStdev":30.5874756,"utility":-0.0126349461,"utilityLcb":-0.796848313,"visits":4,"winrate":0.490970594},{"lcb":0.0932849796,"move":"D16","order":11,"prior":0.0466097519,"pv":["D16","D3","Q17"],"scoreLead":-0.370538535,"scoreMean":-0.370538535,"scoreSelfplay":-0.47277722,"scoreStdev":30.7032725,"utility":-0.0168624515,"utilityLcb":-1.12477591,"visits":4,"winrate":0.488968356},{"lcb":-0.491438841,"move":"C3","order":12,"prior":0.0324178748,"pv":["C3","R16","C16"],"scoreLead":-0.664244846,"scoreMean":-0.664244846,"scoreSelfplay":-0.482832193,"scoreStdev":30.7828552,"utility":-0.0141759727,"utilityLcb":-2.77438059,"visits":3,"winrate":0.494348521},{"lcb":-0.512627207,"move":"C17","order":13,"prior":0.0303994678,"pv":["C17","R16"],"scoreLead":-0.696721703,"scoreMean":-0.696721703,"scoreSelfplay":-0.665783852,"scoreStdev":30.7520999,"utility":-0.0275793309,"utilityLcb":-2.8,"visits":2,"winrate":0.487372793},{"lcb":-0.505080722,"move":"R3","order":14,"prior":0.0199293494,"pv":["R3","R16"],"scoreLead":-0.308975212,"scoreMean":-0.308975212,"scoreSelfplay":-0.199910849,"scoreStdev":30.7235825,"utility":-0.00930808168,"utilityLcb":-2.8,"visits":2,"winrate":0.494919278}],"rootInfo":{"currentPlayer":"B","scoreLead":0.13919092,"scoreSelfplay":0.0775131322,"scoreStdev":30.5802453,"symHash":"47E1EBDAE69A5DED3FF8DB1FA4E91845","thisHash":"47E1EBDAE69A5DED3FF8DB1FA4E91845","utility":0.00410846401,"visits":100,"winrate":0.49834412},"turnNumber":0}
